### DISCUSSION TITLE
ltsp.conf client parameters not found

### DISCUSSION THREAD

**Discussion by wforumw** [ltsp/ltsp#423] [Q&A] [answered]
We have updated ltsp to version 21.01-1~ubuntu18.04.1 installed and run 
`ltsp initrd `
Generated ltsp.img under /srv/tftp/ltsp/ltsp.img

and now when the client boots I see 
```
/sbin/ltsp: line xxx: CUPS_SERVER: not found`
/sbin/ltsp: line xxx: FSTAB_1: not found`
```
In ltsp.conf it is configured like
```
[clients]
CUPS_SERVER       = cupsserver
FSTAB_1                 = "192.168.2.2:/ / nfs4 defaults,vers=4.0,soft,nolock,retrans=5 0 0"
```
If we put back the old image under /srv/tftp/ltsp/ltsp.img it works again

Did something change in the new ltsp version?

**Comment by alkisg** [comment, marked as answer]
Hi, please search for the word "space" in https://ltsp.org/man/ltsp.conf/

This hasn't changed since ltsp 19.x, a single space around = is tolerated, but generally the use of spaces in parameter assignments isn't supported.

In shell, the `CUPS_SERVER      = cupsserver` command means:
Run the "CUPS_SERVER" executable with two parameters, the first parameter is "=" and the second is "cupsserver". It doesn't set an environment variable.

**Reply by wforumw** [reply]
Thanks it works now. The issue was the spaces.
